### PR TITLE
Discover adjustment

### PR DESCRIPTION
**Pull request by zehfernandes**
- Replace banner copy 
- Fix hover issue with fork icon
- Remove featured sandbox component.
- Add a random seed to refresh content per day, not at each visit. 
- Remove link in the last item of the breacrumbs
- Remove drag-and-drop behavior in the whole Discovery page

### CODE REVIEW

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/5884/commits/9961e02b49f834384b08c6dc29ce3e4ba437bf5d is at https://pr5884.build.csb.dev/s/new.

File: packages/app/src/app/pages/Dashboard/Content/routes/Discover/index.tsx [hunk 1]
```diff
@@ -101,14 +105,14 @@ export const Discover = () => {
...
-            paddingY: 10,
+            paddingY: 0,
             userSelect: 'none',
           })}
         >
           <Banner />
 
           <Stack direction="vertical" gap={16}>
-            <FeaturedSandboxes />
+            {/* <FeaturedSandboxes /> */}
```
Comment (siddharthkp) [inline packages/app/src/app/pages/Dashboard/Content/routes/Discover/index.tsx:115]: oooooh, no featured section?

File: packages/app/src/app/pages/Dashboard/Content/utils.ts [hunk 2]
```diff
@@ -39,3 +39,38 @@ export function getPossibleTemplates(
...
+    return x - Math.floor(x);
+  };
+
+  const length = array == null ? 0 : array.length;
+  if (!length) {
+    return [];
+  }
+  let index = -1;
+  const lastIndex = length - 1;
+  const result = copyArray(array);
```
Comment (siddharthkp) [inline packages/app/src/app/pages/Dashboard/Content/utils.ts:68]: Looks like you want a shallow copy (right?), you can also do it like this:

```suggestion
  const result = [ ...array ]
```

Comment (zehfernandes) [reply packages/app/src/app/pages/Dashboard/Content/utils.ts:68]: Do you know if `[...array] works with Proxy object?

**Comment by DannyRuchtie** [comment]
We need better separation from trending.
- more space or separator? 

Love all the other tweaks!

File: packages/app/src/app/pages/Dashboard/Content/utils.ts (hunk 2 above)
Comment (siddharthkp) [reply packages/app/src/app/pages/Dashboard/Content/utils.ts:68]: yep, it should work

**Review by DannyRuchtie — APPROVED** [review]
ship it

**Pull request merged**